R/annotation-data-helper.R

```diff
@@ -109,17 +93,17 @@ getUniqueIntronRanges <- function(intronRangesByTx.unlist) {
 
```

Comment:
New implementation groups data by transcript (`group_name`), and depending on the strand of the transcript, assigns intron rank.